configure.py

```diff
@@ -1802,6 +1831,7 @@ def configure_v8(o, configs):
   o['variables']['v8_enable_external_code_space'] = 1 if options.enable_pointer_compression else 0
   o['variables']['v8_enable_31bit_smis_on_64bit_arch'] = 1 if options.enable_pointer_compression else 0
   o['variables']['v8_enable_extensible_ro_snapshot'] = 0
+  o['variables']['v8_enable_temporal_support'] = 1 if options.v8_enable_temporal_support else 0
```

Comment:
Should we make it imply `v8_enable_i18n_support = 1` too? (probably won't be very useful to try to make v8_enable_i18n_support = 0 + v8_enable_temporal_support = 1 work together anyway). I suppose V8 will keep ICU4U + ICU4X for quite some time.